_usage/migration.md

```diff
@@ -71,7 +71,7 @@ your database connections. Simply make your applications point to the new data
 source and launch your tests.
 
 To a client, Babelfish is expected to look just like a normal SQL Server. Unless
-you hit a [missing feature](/docs/usage/missing-features) your apps are expected to work
+you hit a [limitation](/docs/usage/limitations-of-babelfish), your apps are expected to work
 normally under most circumstances. However, we recommend thorough testing, and
 adjusting details here and there.
 
```

Comment:
Avoid 'here and there' , avoid 'details' -- it is imprecise. Better: "However, we recommend thorough testing of the application after any required changes to SQL code have been made"

Reply:
Again, I see no harm in this. It clearly refers to "Make SQL code adjustments as needed." above.

Reply:
"here and there" simply does not mean nothing anything, so it is best removed. Avoiding unnecessary wording makes every technical text better.

Reply:
Ok, removed.